### DISCUSSION TITLE
all vallue's are inverted

### DISCUSSION THREAD

**Discussion by Donfox95** [firefly-iii/firefly-iii#8777] [Firefly III]
so after importing , all  the revenue account numbers are in negative  and all expense accounts are in positive and i think should be inverted no ?

used the folowing as template for my bank  : 
<img width="1036" alt="image" src="https://github.com/firefly-iii/firefly-iii/assets/106558878/56ee88e5-599d-49f5-a748-2124f14bf590">

example : 
<img width="934" alt="image" src="https://github.com/firefly-iii/firefly-iii/assets/106558878/51437f73-92dc-4e35-bdf0-8318449b2783">

<img width="962" alt="image" src="https://github.com/firefly-iii/firefly-iii/assets/106558878/4f79c9bc-ecaf-4dc8-a1dc-a588089fc13c">

any idea what i did wrong ?

**Comment by JC5** [comment]
This is in fact correct, so no worries. 

Revenue accounts have a negative balance since they send you money. Expense accounts have a positive balance because you send money to them.
